pkg/services/apiserver/aggregator/aggregator.go

```diff
@@ -260,13 +263,23 @@ func CreateAggregatorServer(config *Config, delegateAPIServer genericapiserver.D
 		}
 	}
 
+	metrics := newAvailabilityMetrics()
+
+	// create shared (remote and local) availability metrics
+	// TODO: decouple from legacyregistry
+	registerIntoLegacyRegistryOnce.Do(func() { err = metrics.Register(legacyregistry.Register, legacyregistry.CustomRegister) })
```

Comment:
I copied this from kube-aggregator and not fully sure if this is what causes the duplicate.